Makefile

```diff
@@ -20,3 +21,21 @@ container-push-operator:
 
 container-push-registry:
 	docker push $(IMAGE_REGISTRY)/$(REGISTRY_IMAGE):$(IMAGE_TAG)
+
+operator-sdk:
+	curl -JL https://github.com/operator-framework/operator-sdk/releases/download/$(OPERATOR_SDK_VERSION)/operator-sdk-$(OPERATOR_SDK_VERSION)-x86_64-linux-gnu -o operator-sdk
+	chmod 0755 operator-sdk
+
+_out:
```

Comment:
not sure if its critical here, but maybe delete the directory if exists and re create it (if there is a possible scenario of existing artefacts)

Reply:
Probably, not critical, but won't hurt. Will fix.